### DISCUSSION TITLE
Is it possible to modelise an Multilayer Wall in Cadwork with python ?

### DISCUSSION THREAD

**Discussion by faure-levoux** [cwapi3d/cwapi3dpython#168] [General]
Is it possible to create this with a function in python ? 
From here : ![image](https://github.com/cwapi3d/cwapi3dpython/assets/54621758/9ddf7524-8dc3-466f-961c-a0bb3d16ea16)
To here : 
![image](https://github.com/cwapi3d/cwapi3dpython/assets/54621758/72c227a3-c126-4d84-9904-fad8c15826d4)

Thanks !

**Comment by Brunner246** [comment]
see #103
